chalicelib/action.py

```diff
@@ -350,3 +347,26 @@ def _valid_reason(self, reason: str) -> bool:
             return True
 
         return False
+
+    def _create_list_message(self, data) -> None:
+        """
+        Create the slack block message layout for list action
+        """
+        data = json.loads(data)
+
+        start_date = data[0].get("event_date")
+        end_date = data[-1].get("event_date")
+
+        self.slack.add_section_block(
+            text=f"Reported time for period *{start_date}:{end_date}*",
+        )
+        self.slack.add_divider_block()
+
+        for event in data:
+            start_date = event.get("event_date")
```

Comment:
dont we have this already at the top? `start_date = data[0].get("event_date")`

Reply:
This is per event, but yeah the name of the variable can be better ☺️